File: Java/codist.java
//This program calculates the distance between two points using the Math class.

package Java_Samples;
import java.util.Scanner;

public class codist {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter x1 : ");
        int x1 = sc.nextInt();
        System.out.print("Enter y1 : ");
        int y1 = sc.nextInt();
        System.out.print("Enter x2 : ");
        int x2 = sc.nextInt();
         System.out.print("Enter y2 : ");
        int y2 = sc.nextInt();
        
        double dist = Math.sqrt(Math.pow((x2-x1), 2)+Math.pow((y2-y1), 2));

        System.out.println("Distance between the coordinates are : "+dist);
        sc.close();
    }
    
}

File: C/codist.c
//This program calculates the distance between two points in 2D space.

#include<stdio.h>
#include<math.h>

int main(){
    int x1,x2,y1,y2;

    printf("Enter x1 :");
    scanf("%d",&x1);
    printf("Enter y1 :");
    scanf("%d",&y1);
    printf("Enter x2 :");
    scanf("%d",&x2);
    printf("Enter y2 :");
    scanf("%d",&y2);

    double dist = sqrt(pow(x2-x1,2)+pow(y2-y1,2));
    printf("The distance between the coordinates : %.2lf",dist);

    return 0;
}